### DISCUSSION TITLE
Client sent an HTTP request to an HTTPS server.

### DISCUSSION THREAD

**Discussion by LukeVader-IV** [nextcloud/all-in-one#3503] [Questions] [answered]
### Steps to reproduce
1. Start Nextcloud docker AIO for the first time
2. fill in your valid domain for verification

### Expected behavior 
The domain gets verified and the setup application continues setup

### Actual behavior 
The GUI tells the user there's been an issue, and to check the logs of the mastercontainer for more details.
The logs say the following:
`The response of the connection attempt to "http://{domain}:443" was: Client sent an HTTP request to an HTTPS server.`

### Host OS 
Open Media Vault 6.9.2-1 (Shaitan) / Debian 11

#### Nextcloud AIO version 
7.2.1

#### Current channel 
latest

**Comment by LukeVader-IV** [comment]
PS: no there's no reverse proxy, that domain points straight to the server

**Comment by szaimen** [comment]
Hi, see https://github.com/nextcloud/all-in-one#how-to-skip-the-domain-validation

**Comment by LukeVader-IV** [comment]
Yes i have seen this page, as you seem to reply with this link to anyone with this issue.
While this may be a temporary solution, it's not the solution.
If the intention is not to have it check the domain, why include it at all.
Also, may i ask why it's been closed as a bug report and opened as a "discussion"? This is clearly a bug with Nextcloud AIO, or at least the Docker image.
No program should try making an HTTP request to port 443.
I am unmaking your answer as this isn't a question, it's a bug report.

**Reply by szaimen** [reply]
This is not a bug. Possibly your network is already running a different 443 server with https. That would explaing why the validation does not work.

**Comment by LukeVader-IV** [comment]
Your suggestions did not solve the issue.

The AIO container is being forwarded to port 443 and port 80 externally.
When connecting to the domain externally I can connect to the interface. When connecting to port 80 externally, I get redirected to port 443. On port 443 I get the AIO interface.

The documentation indicated it should automatically figure it out when doing this.
However it doesn't and when inputting the exact same domain as i am currently connected to, it throws an error (see original post).

PS: **do not mark your own answer as an answer.** it is not the answer unless it has worked for me. which will most likely be a patch.

**Comment by szaimen** [comment]
> On port 443 I get the AIO interface.

You found your issue then. The aio interface is intended to use port 8080 or 8443. the apache container is intended to use port 443.

**Reply by szaimen** [reply]
Also see https://github.com/nextcloud/all-in-one#which-ports-are-mandatory-to-be-open-in-your-firewallrouter

**Comment by szaimen** [comment, marked as answer]
Problem solved in https://github.com/nextcloud/all-in-one/discussions/3607
